jax/_src/numpy/ufuncs.py

```diff
@@ -203,19 +203,80 @@ def ceil(x: ArrayLike, /) -> Array:
     return lax.asarray(x)
   return lax.ceil(*promote_args_inexact('ceil', x))
 
-@implements(np.exp, module='numpy')
+
 @partial(jit, inline=True)
 def exp(x: ArrayLike, /) -> Array:
+  """Calculate element-wise exponential of the input.
+
+  JAX implementation of :func:`numpy.exp`.
```

Comment:
I think for ufuncs, we need to use ``` :obj:`numpy.exp` ``` rather than ``` :func:`numpy.exp` ```, otherwise no cross-reference link is generated.

Reply:
Done